### ISSUE TITLE
Set default cropping area?

### ISSUE THREAD

**Issue by kla-ko** [advanced-cropper/react-advanced-cropper#37]
Maybe a simple thing to do, but I couldn't figure it out:
I want to set the default croppping area (the area with the handles) when initializing a cropper. In my case it should be 100% of the visible image on the screen. To make it less easy: I don't know the visible area in pixel in advance, because of a responsive display.
Any suggestion is appreciated ...

Regards,

**Comment by Norserium** [comment]
@kla-ko, hello, nice to see you again! Did you try the approach that described in [this guide](https://advanced-cropper.github.io/react-advanced-cropper/docs/guides/advanced-recipes#fill-the-image) (Fill the image)?

**Comment by kla-ko** [comment]
Yes,  I did. The thing is, ``width`` and ``height`` seem to be in px and I don't know how to get these numbers as these are not fixed. Even if the cropper component has been instantiated, a user can resize the screen and the visible size of the cropper will change.

**Comment by Norserium** [comment]
@kla-ko, do we talk about this example?

```tsx
import React from 'react';
import { Cropper } from 'react-advanced-cropper';

export const Example = () => {
    const defaultSize = ({ imageSize, visibleArea }) => {
            return {
                width: (visibleArea || imageSize).width,
                height: (visibleArea || imageSize).height,
            };
    }
    return (
        <Cropper
            src={'https://images.pexels.com/photos/6524107/pexels-photo-6524107.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940'}
            defaultSize={defaultSize}
        />;
    )
};
```

**Comment by kla-ko** [comment]
Alright! I was fiddeling with defaultCoordinates and didn't recognize this example. Sorry.
As always - very well prepared docu ....
Thanks

**Comment by Norserium** [comment]
@kla-ko, you are welcome!
